frontend/src/pages/FormBCeIDPage.vue

```diff
@@ -370,6 +371,15 @@ const scrollToNewContact = () => {
     });
   }
 };
+
+const districtsList = ref([]);
+useFetchTo("/api/clients/districts?page=0&size=250", districtsList);
```

Comment:
We changed the URL so it's now `/api/districts` instead of `/api/clients/districts`